go.mod

```diff
@@ -4,13 +4,15 @@ go 1.18
 
 require (
 	github.com/caarlos0/env/v6 v6.10.1
+	github.com/gofrs/uuid/v5 v5.3.2
```

Comment:
https://pkg.go.dev/github.com/google/uuid#NewV7
What's the difference between google and gofrs uuid?